policy/aggregation_type.go

```diff
@@ -141,6 +158,40 @@ func (a AggregationType) IsValidForTimer() bool {
 	}
 }
 
+// Quantile returns the quantile represented by the AggregationType.
+func (a AggregationType) Quantile() (float64, bool) {
```

Comment:
Don't really feels like an error, see its usage in the Quantiles() function, I'd rather check the bool there rather than swallowing the error.